docs/reference/indices/aliases.asciidoc

```diff
@@ -3,14 +3,23 @@
 
 APIs in Elasticsearch accept an index name when working against a
 specific index, and several indices when applicable. The index aliases
+<<<<<<< HEAD
```

Comment:
something went wrong here.

Reply:
Yes, I know, and I'm sorry. As a result of being a newbie to both Elastic and Git, I've created a monster that's bent on haunting me forever. Deb has been guiding me on a twisted path toward a solution, and we hope to have it tamed in the next day or two (fingers crossed).

Reply:
hehe... no worries. When you have tamed the git monster,  please teach me too :)